css-scroll-snap-2/Overview.bs

```diff
@@ -262,7 +262,7 @@ Snap Events {#snap-events}
 ████████    ███    ████████ ██    ██    ██     ██████
 -->
 
-{{snapChanged}} and {{snapChanging}} {#snapchanged-and-snapchanging}
+{{snapchanged}} and {{snapchanging}} {#snapchanged-and-snapchanging}
```

Comment:
see [discussion using past tense for the event name](https://github.com/w3c/csswg-drafts/issues/7442#issuecomment-1214626815)